signals.mml

```diff
@@ -110,7 +110,9 @@ Layer:
                 tags->'railway:atb-ng' AS atb_ng,
                 tags->'railway:atb-vv' AS atb_vv,
                 railway_etcs_null_no(tags->'railway:etcs') AS etcs,
-                railway_etcs_null_no(tags->'construction:railway:etcs') AS construction_etcs
+                railway_etcs_null_no(tags->'construction:railway:etcs') AS construction_etcs,
+                tags->'railway:kvb' AS kvb,
+                tags->'railway:tvm' AS tvm
```

Comment:
I would move the new tags before the etcs ones, this would reduce the amount of changed lines (but has no technical difference).

Reply:
Thanks, done !